python/hopsworks/flink_execution.py

```diff
@@ -0,0 +1,348 @@
+        job_type=None,
+    ):
+        self._id = id
+        self._final_status = final_status
+        self._state = state
+        self._submission_time = submission_time
+        self._stdout_path = stdout_path
+        self._stderr_path = stderr_path
+        self._args = args
+        self._progress = progress
+        self._user = user
+        self._duration = duration
+        self._monitoring = monitoring
+        self._app_id = app_id
+        self._hdfs_user = hdfs_user
+        self._job_name = job_name
+        self._job_type = job_type
+        self._project_id = project_id
+
+        self._execution_engine = execution_engine.ExecutionEngine(project_id)
+        self._execution_api = execution_api.ExecutionsApi(project_id)
+        self._flink_cluster_api = flink_cluster_api.FlinkClusterApi(
+            project_id, project_name
+        )
+
+    def get_jobs(self):
+        """Get jobs from the specific execution of the flink cluster.
+        ```python
+
+        # log in to hopsworks
+        import hopsworks
+        project = hopsworks.login()
+
+        # fetch flink cluster handle
+        flink_cluster_api = project.get_flink_cluster_api()
+        flink_cluster = flink_cluster_api.get_cluster(name="myFlinkCluster")
+
+        # get all executions(This will return empty list of no execution is running on this Flink cluster)
+        executions = flink_job.get_executions()
+
+        # select 1st execution
+        execution = executions[0]
+
+        # get jobs from this execution
+        execution.get_jobs()
+        ```
+
+        # Returns
+            `List[Dict]`: The array of dicts with flink job id and and status of the job.
+        # Raises
+            `RestAPIError`: If unable to get the jobs from the execution
+        """
+
+        return self._flink_cluster_api._get_jobs(self)
+
+    def get_job(self, job_id):
+        """Get specific job from the specific execution of the flink cluster.
+        ```python
+
+        # log in to hopsworks
+        import hopsworks
+        project = hopsworks.login()
+
+        # fetch flink cluster handle
+        flink_cluster_api = project.get_flink_cluster_api()
+        flink_cluster = flink_cluster_api.get_cluster(name="myFlinkCluster")
+
+        # get all executions(This will return empty list of no execution is running on this Flink cluster)
+        executions = flink_job.get_executions()
+
+        # select 1st execution
+        execution = executions[0]
+
+        # get jobs from this execution
+        job_id = '113a2af5b724a9b92085dc2d9245e1d6'
+        execution.get_job( job_id)
+        ```
+
+        # Arguments
+            job_id: id of the job within this execution
+        # Returns
+            `Dict`: Dict with flink job id and and status of the job.
+        # Raises
+            `RestAPIError`: If unable to get the jobs from the execution
+        """
+
+        return self._flink_cluster_api._get_job(self, job_id)
+
+    def stop_execution(self):
```

Comment:
This function does the same thing as `stop()`? I think we should just keep the `stop()` function and remove the `stop_execution()` one